interface/forms/procedure_order/common.php

```diff
@@ -0,0 +1,1287 @@
+                                        echo "<a class='btn btn-outline-primary' target='_blank' href='$rootdir/procedure_tools/clarity/ereq_form.php?debug=1&formid=$formid'>" . xlt("Manual eREQ") . "</a>";
+                                    }
+                                    ?>
+                                </div>
+                            </div>
+                        </div>
+                        <div class="col-md-12">
+                            <legend class="bg-dark text-light"><?php echo xlt('Order Log'); ?></legend>
+                            <div class="jumbotron m-0 px-2 py-0 overflow-auto" id="processLog" style="max-height:500px;">
+                                <?php
+                                if (!empty($order_log)) {
+                                    $alertmsg = text($order_log);
+                                } else {
+                                    $order_log = text($alertmsg);
+                                }
+                                if (!empty($alertmsg)) {
+                                    echo nl2br(text($alertmsg));
+                                }
+                                ?>
+                                <input type="hidden" name="order_log" value="<?php echo text($order_log); ?>">
+                            </div>
+                        </div>
+                    </div>
+                </div>
+                <div class="row form-group clearfix">
+                    <div class="float-left position-override mt-2">
+                        <div class="btn-group" role="group">
+                            <button type="submit" class="btn btn-primary btn-save"
+                                name="bn_save" id="bn_save" value="save"
+                                onclick='transmitting = false;'><?php echo xlt('Save Current'); ?>
+                            </button>
+                            <button type="submit" class="btn btn-success btn-save"
+                                name='bn_save_exit' id='bn_save_exit' value="save_exit"
+                                onclick='transmitting = false;'><?php echo xlt('Save'); ?>
+                            </button>
+                            <button type="submit" class="btn btn-primary btn-transmit"
+                                name='bn_xmit' value="transmit"
+                                onclick='transmitting = true;'><?php echo xlt('Transmit Order'); ?>
+                            </button>
+                            <button type="button" class="btn btn-secondary btn-cancel"
+                                onclick="top.restoreSession();location='<?php echo $GLOBALS['form_exit_url']; ?>'"><?php echo xla('Cancel/Exit'); ?>
+                            </button>
+                        </div>
+                        <span class="wait fa fa-cog fa-spin fa-2x ml-2 d-none"></span>
+                    </div>
+                </div>
```

Comment:
gulp, that was a lot of code :)
Guessing you were bringing in older scripts into new ones, which is why I noted some escaping/translation issues (ie. you got most of them, but some of them creeped through :) ).